Fix pre-main-skr-kyma-eventing (#13574)
kyma-project/kyma: tests/fast-integration/eventing-test/README.md

COMPASS_TENANT=              # Only required if KYMA_TYPE=SKR
GARDENER_KUBECONFIG=         # Only required if KYMA_TYPE=SKR
```
>**IMPORTANT:** The `TEST_SUFFIX` is required for every test run for SKR cluster. It needs to be 4 characters long, as it is the name for the compass scenario.
> The eventing tests are adding a scenario with the `test-TEST_SUFFIX` name and if you did not clean the compass scenario (which is done in the `eventing-test-cleanup.js` script),

> > the tests will try to reuse an already existing scenario und will fail on the tests preparation with the following error: *Update of API/Event is not supported yet for the compass scenario*.

Author: I added a script which lists the registered scenarios and also a troubleshooting guide how to avoid this error

> > the tests will try to reuse an already existing scenario und will fail on the tests preparation with the following error: *Update of API/Event is not supported yet for the compass scenario*.

Reviewer: Thank you! Please fix the `und` with `and` :) in line `already existing scenario und will fail on the tests`.

> > the tests will try to reuse an already existing scenario und will fail on the tests preparation with the following error: *Update of API/Event is not supported yet for the compass scenario*.

Author: Done

COMPASS_TENANT=              # Only required if KYMA_TYPE=SKR
GARDENER_KUBECONFIG=         # Only required if KYMA_TYPE=SKR
```
>**IMPORTANT:** The `TEST_SUFFIX` is required for every test run for SKR cluster. It needs to be 4 characters long, as it is the name for the compass scenario.

> > The eventing tests are adding a scenario with the `"test-TEST_SUFFIX"` name and if you did not clean the compass scenario (which is done in the `eventing-test-cleanup.js` script),

Reviewer 2: ```suggestion
> The eventing tests add a scenario with the `"test-TEST_SUFFIX"` name. 
> You must clean the compass scenario, which is done in the `eventing-test-cleanup.js` script. If you did not clean it, the tests try to reuse an existing scenario and will fail during the tests preparation with the following error: *Update of API/Event is not supported yet for the compass scenario*.
```

Trying to split that very long sentence. Also trying to avoid that a sentence is split across two lines.

Author: Updated. I actually even removed the second part of the second sentence, as it is covered in the troubleshooting part.

> > the tests will try to reuse an already existing scenario and will fail on the tests preparation with the following error: *Update of API/Event is not supported yet for the compass scenario*.

Reviewer 2: ```suggestion
```

Author: https://github.com/kyma-project/kyma/pull/13574#discussion_r832072023

>**NOTE:** The Event Mesh Credentials JSON file can be downloaded from the BTP Cockpit under your subaccount instances.

3. Run the following command to set up the environment variables in your system:
```bash
export $(xargs < .env)
```

> 4. Execute the Eventing tests locally

Reviewer 2: ```suggestion
4. Execute the Eventing tests locally:
```

Author: done

- **inside the OSS cluster:**
```bash
npm run test-eventing
```
- **inside the SKR cluster:**
```bash
npm run test-eventing TEST_SUFFIX=abcd
```

## Troubleshooting ##

> 1. if the `TEST_SUFFIX` env variable was not set during the test execution or a scenario with that suffix already exists, you can get the following error:

Reviewer 2: "can" = maybe? or definitely? If definitely, then drop it

Why the numbering? Where is number 2 ?

```suggestion
If the `TEST_SUFFIX` environment variable was not set during the test execution or a scenario with that suffix already exists, you get the following error:
```

Author: done

>_Update of API/Event is not supported yet for the compass scenario_.

> To avoid this error you can first get the list of the already used scenarios:

Reviewer 2: ```suggestion
To avoid this error, first get the list of the existing scenarios:
```

Author: done

```bash
 npm run eventing-get-registered-scenarios
```

> and then assign the `TEST_SUFFIX` env variable a value which is not listed in the output of the command above.

Reviewer 2: ```suggestion
Then, for the `TEST_SUFFIX` environment variable, assign a value that is not listed in the list of existing scenarios.
```

Author: done
